cypher-shell/src/main/java/org/neo4j/shell/cli/CliArgHelper.java

```diff
@@ -81,6 +82,8 @@ public static CliArgs parse(@Nonnull String... args) {
 
         cliArgs.setEncryption(ns.getBoolean("encryption"));
 
+        cliArgs.setDebugMode(ns.getBoolean("debug"));
```

Comment:
I think the `debug` functionality fundamentally alters the state of how runners execute statements. Doing this over static method seems that it could change the state anywhere, ideally I'd like it to be done as part of a constructor instead of a static method. Thereby making Helper a class with states instead of a collection of static methods.